src/storage/engine/mod.rs

```diff
@@ -324,7 +326,7 @@ impl Cursor {
         }
 
         statistics.seek += 1;
-
+        self.seek_last_time = true;
```

Comment:
Should we also mark `last_op_is_seek` in `near_seek`?

Reply:
no, we only set it true before following operates:
1. self.iter.seek
2.self.iter.seek_for_prev
3. self.iter.seek_to_last
4. self.iter.seek_to_first

and set it to false before following operates:
1. self.iter.next
2.self.iter.prev

@zhangjinpeng1987